numpy/core/src/umath/ufunc_object.c

```diff
@@ -3927,7 +3928,20 @@ PyUFunc_GenericReduction(PyUFuncObject *ufunc, PyObject *args,
                      _reduce_type[operation]);
         return NULL;
     }
-
+    /* if there is a tuple of 1 for `out` in kwds, unpack it */
+    if (kwds != NULL) {
+        PyObject *out_obj = PyDict_GetItem(kwds, npy_um_str_out);
+        if (out_obj != NULL && PyTuple_Check(out_obj)) {
```

Comment:
OK, I'll change that, here and in the two other locations where `out` is treated exactly in the same way (once in this file, once in `override.c`).